release/release.go

```diff
@@ -30,38 +30,41 @@ import (
 // Series holds the Ubuntu Core series for snapd to use.
 var Series = "16"
 
-// LSB contains the /etc/lsb-release information of the system.
+// LSB contains the /etc/os-release information of the system.
 type LSB struct {
```

Comment:
Please add comments as to how those variables map to /etc/os-release (or follow up with identical naming later)